src/main/resources/db/migration/V1_create_movies.sql

```diff
@@ -0,0 +1,9 @@
+DROP TABLE IF EXISTS movie CASCADE;
+CREATE TABLE movie
+(
+    id         INTEGER GENERATED BY DEFAULT AS IDENTITY,
```

Comment:
É importante alterar a coluna ID para manter o padrão e também o datatype. Segue abaixo o exemplo:

"ID_MOVIE NUMBER (24, 0) GENERATED BY DEFAULT ON NULL AS IDENTITY START WITH 1 INCREMENT BY 1 MINVALUE 0 NOMAXVALUE NOT NULL,"